devel/xapian-core/Portfile

```diff
@@ -104,15 +104,28 @@ if [string match {xapian-bindings-*} ${subport}] {
 
 # Java
 subport xapian-bindings-java {
+    PortGroup                   java 1.0
+
     revision                    0
 
     description                 Xapian bindings for Java
     long_description            ${description}
 
+    # Required java version
+    java.version                6+
+    # JDK port to install if required java not found
+    java.fallback               openjdk11
+
     configure.args-replace      --without-java --with-java
 
-    # TODO: libxapian_jni.jnilib gets "installed" into the work directory:
-    # https://trac.xapian.org/ticket/774
```

Comment:
I would keep the reference to the upstream ticket in the Portfile, to explain why we have to move the file from its weird installation directory, and so that we can periodically check it to see if upstream ever improves the situation.